### PR TITLE
Simplify formify logic

### PR DESCRIPTION
**Pull request by jeffsee55**

### CODE REVIEW

**Comment by logan-anderson** [comment]
@jeffsee55  One thing I noticed is now on the iframe starter we the Global form is displayed by default.

Screenshot 2023-04-18 at 8 43 01 AM

**Comment by jeffsee55** [comment]
Thanks @logan-anderson , fixed.

File: packages/@tinacms/app/package.json [hunk 1]
```diff
@@ -12,24 +12,25 @@
...
     "react-router-dom": "6.3.0",
-    "tinacms": "workspace:*",
-    "xstate": "4.32.1",
     "tailwindcss": "^3.2.7",
+    "tinacms": "workspace:*",
     "typescript": "^4.6.4",
-    "@monaco-editor/react": "4.4.5",
-    "monaco-editor": "0.31.0",
-    "webfontloader": "1.6.28"
+    "vite": "3.1.8",
```
Comment (logan-anderson) [inline packages/@tinacms/app/package.json:32]: Is `vite` and `webfontloader` being used in `@tinacms/app` ?  I did a quick search in the codebase and didn't see any uses.

**Comment by logan-anderson** [comment]
Other then that small dep issue it looks good to me.

**Pull request merged**